2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/.config/wandb/settings
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_setup.py:_flush():75] Loading settings from /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/settings
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_setup.py:_flush():75] Loading settings from environment variables: {}
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_init.py:_log_setup():386] Logging user logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220302_091847-2g8m30x6/logs/debug.log
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_init.py:_log_setup():387] Logging internal logs to /opt/ml/code/git_bcnlp/special_mission/hyunah/wandb/run-20220302_091847-2g8m30x6/logs/debug-internal.log
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_init.py:_jupyter_setup():336] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f3030366af0>
2022-03-02 09:18:47,206 INFO    MainThread:21034 [wandb_init.py:init():420] calling init triggers
2022-03-02 09:18:47,207 INFO    MainThread:21034 [wandb_init.py:init():425] wandb.init called with sweep_config: {}
config: {}
2022-03-02 09:18:47,207 INFO    MainThread:21034 [wandb_init.py:init():471] starting backend
2022-03-02 09:18:47,207 INFO    MainThread:21034 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-03-02 09:18:47,221 INFO    MainThread:21034 [backend.py:ensure_launched():219] starting backend process...
2022-03-02 09:18:47,236 INFO    MainThread:21034 [backend.py:ensure_launched():224] started backend process with pid: 21073
2022-03-02 09:18:47,249 INFO    MainThread:21034 [wandb_init.py:init():480] backend started and connected
2022-03-02 09:18:47,266 INFO    MainThread:21034 [wandb_run.py:_label_probe_notebook():921] probe notebook
2022-03-02 09:18:47,266 INFO    MainThread:21034 [wandb_run.py:_label_probe_notebook():931] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-03-02 09:18:47,266 INFO    MainThread:21034 [wandb_init.py:init():550] updated telemetry
2022-03-02 09:18:47,293 INFO    MainThread:21034 [wandb_init.py:init():581] communicating current version
2022-03-02 09:18:48,086 INFO    MainThread:21034 [wandb_init.py:init():586] got version response upgrade_message: "wandb version 0.12.11 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-03-02 09:18:48,087 INFO    MainThread:21034 [wandb_init.py:init():596] communicating run to backend with 30 second timeout
2022-03-02 09:18:48,300 INFO    MainThread:21034 [wandb_init.py:init():624] starting run threads in backend
2022-03-02 09:18:51,200 INFO    MainThread:21034 [wandb_run.py:_console_start():1827] atexit reg
2022-03-02 09:18:51,201 INFO    MainThread:21034 [wandb_run.py:_redirect():1701] redirect: SettingsConsole.WRAP
2022-03-02 09:18:51,201 INFO    MainThread:21034 [wandb_run.py:_redirect():1738] Wrapping output streams.
2022-03-02 09:18:51,203 INFO    MainThread:21034 [wandb_run.py:_redirect():1762] Redirects installed.
2022-03-02 09:18:51,203 INFO    MainThread:21034 [wandb_init.py:init():651] run started, returning control to user process
2022-03-02 09:18:51,206 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:51,206 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:51,210 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:51,210 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:51,211 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:51,215 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:51,222 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:51,222 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:51,226 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:51,234 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:51,234 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:51,239 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:54,233 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:54,234 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:54,237 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:54,238 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:54,238 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:54,242 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:18:54,244 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:18:54,244 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:18:54,248 INFO    MainThread:21034 [wandb_init.py:_resume_backend():308] resuming backend
2022-03-02 09:22:44,670 INFO    MainThread:21034 [wandb_init.py:_pause_backend():299] pausing backend
2022-03-02 09:22:44,670 INFO    MainThread:21034 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-03-02 09:24:28,460 INFO    MainThread:21034 [wandb_run.py:_atexit_cleanup():1797] got exitcode: 0
2022-03-02 09:24:28,462 INFO    MainThread:21034 [wandb_run.py:_restore():1769] restore
2022-03-02 09:24:28,709 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-02 09:24:28,917 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 879
}

2022-03-02 09:24:29,360 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 3
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 3703
}

2022-03-02 09:24:29,463 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4029
}

2022-03-02 09:24:29,565 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 879
  total_bytes: 4029
}

2022-03-02 09:24:29,667 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:29,769 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:29,870 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:29,972 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:30,073 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:30,174 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

2022-03-02 09:24:30,715 INFO    MainThread:21034 [wandb_run.py:_wait_for_finish():1929] got exit ret: file_counts {
  wandb_count: 4
}
pusher_stats {
  uploaded_bytes: 4029
  total_bytes: 4029
}

